tests/log.txt_ncdm

```diff
@@ -0,0 +1,380 @@
+[ 0000000.0000 ]: ncdm reference redshift = 9
```

Comment:
Also git rm --cached.

use git add -i  instead of git add -u to add files. git add -i gives you an interface to select which files to 'update'.